api/Cargo.toml

```diff
@@ -1,28 +1,28 @@
 [package]
 name = "reinfer-client"
 version = "0.11.0"
-description = "API client for Re:infer"
+description = "API client the Re:infer conversational data intelligence platform"
```

Comment:
Not sure what you were going for here but this is an odd sentence.

"API client **for Re:infer** the conversational ..." maybe?

Reply:
Ah yes, well spotted